docs/pages/reference/access-controls/roles.mdx

```diff
@@ -409,6 +409,50 @@ spec:
       where: '!contains(session_tracker.participants, user.metadata.name)'
 ```
 
+## Access Requests
+
+The following role fields configure Access Requests. For a full description of
+Access Request configuration fields and their relationship to one another, see
+the [Access Request configuration
+guide](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx).
+
+### Creating Access Requests
+
+Within a role, the `request` field configures a user's ability to create Access
+Requests. The `request` field can fall under both `spec.allow` and `spec.deny`:
+
+|Role Field|Description|Further Information|
+|---|---|---|
+| `request.roles` |Teleport roles that a user can or cannot request|[Restrict role requests](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#restrict-role-requests)|
+| `request.claims_to_roles`|Teleport roles that a user can or cannot access based on their traits.|[Restrict role requests](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#restrict-role-requests)|
+| `request.search_as_roles`|Teleport roles that a user can or cannot assume while searching for resources to request access to.|[Restrict resource requests](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#restrict-resource-requests)|
+| `request.max_duration`|The maximum duration of elevated privileges if an Access Request were granted. | [How long access lasts](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#how-long-access-lasts) |
+| `request.reason`|Configures the behavior of Access Request reasons. | [Requiring request reasons](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#requiring-request-reasons) | 
+| `request.thresholds`|Configures the criteria that an Access Request must meet before Teleport approves or denies it.| [Review thresholds](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#review-thresholds) |
```

Comment:
```suggestion
| `request.thresholds`|Configures the criteria that an Access Request must meet.| [Review thresholds](../../admin-guides/access-controls/access-requests/access-request-configuration.mdx#review-thresholds) |
```

Teleport doesn't approve or deny anything here. And since we're about to release a feature where Teleport _does_ automatically approve some requests I think it best to avoid conflating the two.